src/test/java/core/basesyntax/dao/FileReaderCsvImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileReaderCsvImplTest {
+    private static final String FILENAME = "src/test/resources/testReaderFile.csv";
+    private static final String NON_EXISTENT_FILENAME =
+                                "src/test/resources/testReaderNonExistentFile.csv";
+
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+
+    private final FileReader fileReader = new FileReaderCsvImpl();
+
+    @Test
+    public void readFromGoodFile_Ok() throws Exception {
+        String expectedLines = "type,fruit,quantity\n"
+                + "b,banana,20\n"
+                + "b,apple,100\n"
+                + "s,banana,100\n"
+                + "p,banana,13\n"
+                + "p,apple,10\n"
+                + "s,apple,20\n"
+                + "p,banana,5\n"
+                + "s,banana,50";
+
+        try (FileWriter fileWriter = new FileWriter(FILENAME)) {
+            fileWriter.write(expectedLines);
+        } catch (IOException e) {
+            throw new IOException("Can't write file to " + FILENAME, e);
+        }
+
+        List<String> lines = fileReader.readFromFile(FILENAME);
+        String actualLines = String.join("\n", lines);
+
+        assertEquals("Expected lines from file don't equals read readFromFile method",
+                expectedLines, actualLines);
+    }
+
+    @Test
+    public void readFromEmptyFile_Ok() throws Exception {
```

Comment:
```suggestion
    public void readFromFile_emptyFile_Ok() throws Exception {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.

fix naming in all tests